README.md

```diff
@@ -1,47 +1,63 @@
-# stsci_rtd_theme
-STScI branded sphinx theme that inherits from sphinx_rtd_theme
+# `stsci-rtd-theme`
 
-This theme is based off of the sphinx_rtd_theme, from which it inherits, but the CSS styling
-has been altered for easier reading on a variety of platforms and browsers.
+This is a Sphinx theme branded for the Space Telescope Sciences Institute (STScI).
+
+![Example](stsci_rtd_theme_example.png)
+
+This theme inherits from `sphinx-rtd-theme`; however, the CSS styling has been altered for easier reading on a variety
+of platforms and browsers.
 
 ## Installation
-You can apply this theme to your current documents by installing this repository like any other python package:
 
-Either from GIT:
-```
-git clone https://github.com/spacetelescope/stsci_rtd_theme.git
-python setup.py install
-```
-Or with PIP:
+You can apply this theme to your current documents by installing like any other Python package:
+
+```shell
+pip install stsci-rtd-theme
 ```
-# install directly from the git repository
+
+Alternatively, you may install the latest commit directly from GitHub:
+
+```shell
 pip install -e https://github.com/spacetelescope/stsci_rtd_theme.git
+```
+
+or clone the source code yourself:
```

Comment:
i would maybe put two install options here - cloning the repo makes more sense to me if you are installing it in editable mode (pip install -e .) because otherwise it doesn't really need to be cloned, but if you want the latest changes on master you can pip install from github (pip install git+https://github.com/spacetelescope/stsci_rtd_theme)

Reply:
ok, should installing from PyPI (`pip install stsci-rtd-theme`) be discouraged then?

Reply:
dont discourage install from pypi, pypi install is needed to install the released version for docs using this theme